codegen/src/main/twirl/templates/JavaClient/Client.scala.txt

```diff
@@ -36,31 +36,30 @@ import scala.concurrent.Future;
 import scala.concurrent.Promise;
 
 
-public final class @{service.name}Client implements @{service.name}, AkkaGrpcClient {
-
-  private final ClientState clientState;
-  private final GrpcClientSettings settings;
-  private final CallOptions options;
-  private final Materializer mat;
-  private final ExecutionContext ec;
+public abstract class @{service.name}Client extends @{service.name}LiftedClient implements @{service.name}, AkkaGrpcClient {
+  public static final @{service.name}Client create(GrpcClientSettings settings, Materializer mat, ExecutionContext ec) {
+    return new Raw@{service.name}Client(settings, mat, ec);
+  }
 
+  protected final static class Raw@{service.name}Client extends @{service.name}Client {
```

Comment:
You called it `Default` before - that sounded a bit less scary than 'raw'. Why the change?

Reply:
I've juggled three different names as I toyed with ideas. No preference for any of them. `Default` is good.